components/public-api-server/pkg/apiv1/tokens_test.go

```diff
@@ -71,6 +72,21 @@ func TestTokensService_CreatePersonalAccessTokenWithoutFeatureFlag(t *testing.T)
 		require.Equal(t, connect.CodeInvalidArgument, connect.CodeOf(err))
 	})
 
+	t.Run("invalid argument when name does not match required regex", func(t *testing.T) {
+		_, _, client := setupTokensService(t, withTokenFeatureDisabled)
```

Comment:
Why is the token service configured with the token feature disabled? Not only this test but some others here too.

Reply:
Probably copy paste. It happens to work because the validation happens irrespective of whether the feature is enabled (we only check the feature enabled after we validate we got sensible arguments). I'll follow-up on this (in another PR) to reduce the confusion